src/plugins/intel_gpu/src/plugin/ops/dft.cpp

```diff
@@ -27,27 +27,38 @@ void createDft(ProgramBuilder& p,
     const auto inputs = p.GetInputInfo(op);
     const auto layer_name = layer_type_name_ID(op);
     const auto& friendly_name = op->get_friendly_name();
-    const auto& out_shape = op->get_output_shape(0);
-
-    auto axes_constant = std::dynamic_pointer_cast<ov::op::v0::Constant>(op->get_input_node_shared_ptr(1));
-    OPENVINO_ASSERT(axes_constant != nullptr, "[GPU] Unsupported parameter nodes type in ", friendly_name, " (", op->get_type_name(), ")");
-    auto axes = axes_constant->cast_vector<int64_t>();
-    uint8_t axis_correction = static_cast<uint8_t>(op->get_input_shape(0).size());
-    if (direction != cldnn::dft_direction::forward || mode != cldnn::dft_mode::real) {
-        --axis_correction;
-    }
-    ov::util::normalize_axes(op.get(), axis_correction, axes);
 
-    std::vector<int64_t> signal_size;
-    if (op->get_input_size() == 3) {
-        auto signal_size_constant = std::dynamic_pointer_cast<ov::op::v0::Constant>(op->get_input_node_shared_ptr(2));
-        OPENVINO_ASSERT(signal_size_constant != nullptr, "[GPU] Unsupported parameter nodes type in ", friendly_name, " (", op->get_type_name(), ")");
-        signal_size = signal_size_constant->cast_vector<int64_t>();
+    if (p.use_new_shape_infer()) {
+        if (op->get_input_size() == 3) {
```

Comment:
In case of dynamic model, is there no need to support the case where `axes` and `signal_size` of DFT are constant?